src/lib/Microsoft.Health.Fhir.Ingest/Service/FhirHealthService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Ingest.Service
+{
+    public abstract class FhirHealthService
+    {
+        public abstract Task<FhirHealthCheckStatus> CheckHealth();
```

Comment:
This should probably be updated to take a cancellation token since this will be asynchronous.  You can set it to default if you would like but it should be an option.

Reply:
Updated